integration-libs/cdc/components/register/facade/facade-providers.ts

```diff
@@ -0,0 +1,31 @@
+import { Provider } from '@angular/core';
+import { UserEmailService, UserPasswordService, UserProfileService } from '@spartacus/user/profile/core';
+import {
+  UserEmailFacade,
+  UserPasswordFacade,
+  UserProfileFacade,
+  UserRegisterFacade
+} from '@spartacus/user/profile/root';
+import { CDCRegisterComponentService } from 'integration-libs/cdc/components/register/facade/cdc-register-component.service';
+export const facadeProviders: Provider[] = [
+  UserEmailService,
+  UserPasswordService,
+  UserProfileService,
+  CDCRegisterComponentService,
+  {
+    provide: UserEmailFacade,
+    useExisting: UserEmailService,
+  },
+  {
+    provide: UserPasswordFacade,
+    useExisting: UserPasswordService,
+  },
+  {
+    provide: UserProfileFacade,
+    useExisting: UserProfileService,
+  },
+  {
+    provide: UserRegisterFacade,
+    useExisting: CDCRegisterComponentService,
+  },
+];
```

Comment:
We shouldn't re-declare all those facade's providers. They are already defined in the original User Profile feature. 

Morevoer my intention was that `CDCRegisterComponentService` should overwrite the `RegiserComponentService` but not the global facade (`UserRegisterService`/`UserRegisterFacade`).

The `CDCRegisterComponentService` should overwrite `RegiserComponentService` on the CMS-component mapping level - similar as `CdcLoginFormComponentService` overwrites `LoginFormComponentService` on the CMS-component mapping level.
See diagram https://github.com/SAP/spartacus/pull/15785#discussion_r884857549

Reply:
hi @Platonn,

yes it makes sense, I was trying to follow the structuring of the core library modules for login / registration. Will add the `CDCRegisterComponentService` to overwrite `RegiserComponentService` CMS-component mapping.

thanks